web/src/app/api/[homepage]/route.ts

```diff
@@ -15,15 +15,19 @@ export async function GET() {
 
 	const homePageData = (await res.json()) as SingleType;
 
+	if (!homePageData.data.attributes) {
+		return Response.json("Homepage contains no Data");
+	}
 	const news = homePageData.data.attributes.body
 		.map((el, i) => {
```

Comment:
Small input. You could use a flatMap instead of map. Then you don't need the filter function :) Just do if ... else return []

Reply:
Did not thought about that. Thanks!

Done ;)